src/apis/nimbus.js

```diff
@@ -35,13 +35,38 @@ var nimbus = class extends ExtensionAPI {
     return {
       experiments: {
         nimbus: {
-          async enrollInExperiment(jsonData) {
+          async enrollInExperiment(jsonData, forceEnroll) {
             try {
+              const experimentStore = ExperimentManager.store.getAll();
```

Comment:
I don't think we need to do this and calling it the experimentStore is not quite accurate, since its just the list of recipes